src/components/Touchable/index.js

```diff
@@ -22,7 +22,7 @@ const styles = {
   `,
 }
 
-const BaseTouchable = ({
+export const BaseTouchable = ({
```

Comment:
Maybe move the propTypes to Touchable instead ?

Reply:
I did a const for propTypes and defaultProps :)